docker/db.Dockerfile

```diff
@@ -18,12 +18,18 @@ RUN ["/usr/local/bin/docker-entrypoint.sh", "postgres"]
 # ===================
 # --- Final Image ---
 # ===================
-FROM postgres:14.5
+FROM postgres:14.6
 LABEL maintainer="IETF Tools Team <tools-discuss@ietf.org>"
 
 COPY --from=builder /data $PGDATA
 
 ENV POSTGRES_PASSWORD=password
 ENV POSTGRES_USER=postgres
 ENV POSTGRES_DB=app
-ENV POSTGRES_HOST_AUTH_METHOD=trust
\ No newline at end of file
+ENV POSTGRES_HOST_AUTH_METHOD=trust
+
+# ================================
+# --- Final Image with out data---
+# ================================
+FROM postgres:14.6 AS db-local
+LABEL maintainer="IETF Tools Team <tools-discuss@ietf.org>"
```

Comment:
Not sure I understand the goal of this `db-local` step?

If it's only an empty postgres database, then you should use the postgres image directly? This also adds an unnecessary build step to the `ci-run-tests` above.

Reply:
There are two uses of the `db-local`:
1. For the tests in ci-run-tests
2. To run locally, because not everyone has access to a database snapshot.

The reason for adding a line here is the PostgreSQL version is only defined in this file.
I guess this could be refactored to start with the `db-local` as the base for other variations.